bin/enable_printfs

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
This file is useful, but I'd argue this is also out of scope for the PR. Is this worked into the build flow at all? That would be a nice feature to add.

Reply:
i think its in the scope of this pr. this is my "verilator enhancements" pr. i'm removed the build-script stuff

Reply:
this script is a hack. this could be done in a firrtl pass, and hopefully the firrtl-pass would not be too slow compared to this hacky solution.

however, the firrtl pass does not exist, so we must live with this hack for now

Reply:
The PR title is `Multithread verilator`. This is what goes into the release notes, merge commit, etc. If this is a `Verilator enhancement` PR you need to adjust the title/description.

Reply:
I tend to agree with John here. I think the multi-threading work will take significantly longer to get in/reviewed, whereas the printf stuff would be a quick merge.

Reply:
PR scope adjusted.

Reply:
on the other hand, it is nice to not have to invoke the JVM and run all the firrtl passes again just to change which printfs get displayed. So I can see a case for having the printf filtering happen after the `make verilog` target completes.

Although I admit again, that the current solution is a hack.

Reply:
Why would you need to invoke the JVM? Just have another intermediate make target that runs this script